<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>521</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Action</string>
    </property>
    <widget class="QPushButton" name="pushButtonNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_plus.png</normaloff>img/ic_plus.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInsert">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_insert.png</normaloff>img/ic_insert.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_update.png</normaloff>img/ic_update.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_delete.png</normaloff>img/ic_delete.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Student Details:</string>
    </property>
    <widget class="QLabel" name="labelAvatar_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Code:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Age:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Intro:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditId">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAge">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIntro">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCode_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>Avatar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAvatar">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>240</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Avatar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_update.png</normaloff>img/ic_update.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveAvatar">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>240</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Avt</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/ic_delete.png</normaloff>img/ic_delete.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>List of Student</string>
    </property>
    <widget class="QTableWidget" name="tableWidgetStudent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>141</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Age</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Quản lý sinh viên&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
